### PR TITLE
Remove redundant taxCutoffLine support

### PR DESCRIPTION
**Pull request by kolyose**
In order to remove the property from BE we have to remove it from the SDK first.

Corresponds to a ticket https://app.zenhub.com/workspaces/kontist-engineering-5be06ab44b5806bc2bf14eca/issues/kontist/figure-app/3581#issuecomment-629274380

### CODE REVIEW

**Comment by kolyose** [comment]
yeah, I just noticed this 🤦 removing it now

**Comment by kolyose** [comment]
Reverted back the changes in the schemas since they are going to be autogenerated back anyways

File: lib/graphql/schema.flow.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ export type Scalars = {|
 
 /** The bank account of the current user */
 export type Account = {|
-   __typename?: 'Account',
```
Comment (hihuz) [inline lib/graphql/schema.flow.js:17]: We still have a number of apparently unrelated changes here 🤔 But if you had them when running the `schema:generate` command, all good

Comment (kolyose) [reply lib/graphql/schema.flow.js:17]: yes, that's what it generated 🤷‍♂️

**Pull request merged**